Dockerfile

```diff
@@ -19,9 +28,7 @@ ARG go_opts
 
 RUN apk add --no-cache git
 
-RUN  VERSION=$(git describe --all --exact-match `git rev-parse HEAD` | grep tags | sed 's/tags\///') \
-    && GIT_COMMIT=$(git rev-list -1 HEAD) \
-    && env $go_opts CGO_ENABLED=0 go build \
+RUN env $go_opts CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build \
```

Comment:
Why is `go_opts` here, and then you aren't passing it in anywhere?

Reply:
We use it in the Makefile, and I was wondering if people would be using it for local builds, but yes I guess probably better to remove it, will cause confusion.